rasa_nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -113,7 +113,7 @@ def _load_flag_if_tokenize_intents(self):
         self.intent_tokenization_flag = self.component_config[
```

Comment:
Concerning L33: I'd rather do something like this: https://github.com/keras-team/keras/blob/master/keras/utils/vis_utils.py#L10

So importing, if it fails, set to `None` and then have a separate function like this https://github.com/keras-team/keras/blob/master/keras/utils/vis_utils.py#L16 to see if the import worked when it is needed (I think a good place for that check would be __init__ of this class).